web/items.go

```diff
@@ -0,0 +1,88 @@
+package web
+
+import (
+	"log"
+	"net/http"
+
+	"time"
+
+	"github.com/btshopng/btshopng/config"
+	"github.com/btshopng/btshopng/models"
+	uuid "github.com/satori/go.uuid"
+)
+
+func NewItemHandler(w http.ResponseWriter, r *http.Request) {
+
+	data, err := GetProfileData(r)
```

Comment:
This is not a good way of doing that. And its not ideal to be passing a http.Request body into a function. I hope you know how io.ReadWriters like r.Body work? When you read from them once, the data isnt available again, since its a stream. 

and the getProfile data should simply be calling a model. eg 

`
user := models.User{}
user.Get(config.Get(),'userid)

`